tests/test_on_lab_rat.rs

```diff
@@ -53,6 +56,9 @@ fn test_modify_lab_rat_memory() {
         .expect("Failed to read line from stdout");
     print!("Lab Rat says: {}", line);
 
+    // Add a delay here
+    thread::sleep(Duration::from_secs(1));

```

Comment:
Was it not working without the delay? Synchronization via sleep often leads to flaky tests

Reply:
Added while troubleshooting, not needed. Removed it.